packages/gateway/src/graphql/query-generator.ts

```diff
@@ -48,6 +49,8 @@ export function generateQueryForType(
         fieldStr += `${fieldName} { ${buildFields(fieldType)} } `
       } else if (isScalarType(fieldType)) {
         fieldStr += `${fieldName} `
+      } else if (isNamedType(fieldType)) {
+        fieldStr += `${fieldName} `
```

Comment:
@rikukissa this was missing enums, that's why the `packages/gateway/src/features/registration/__snapshots__/type-resolvers.test.ts.snap` test had to be updated